crates/sui-rpc-loadgen/src/payload/rpc_command_processor.rs

```diff
@@ -201,6 +241,19 @@ impl Processor for RpcCommandProcessor {
                     divide_checkpoint_tasks(&clients, data, config.num_threads).await
                 }
             }
+            CommandData::QueryTransactionBlocks(data) => {
+                self.load_cache_from_file(CacheType::SuiAddress);
+                if !config.divide_tasks {
+                    vec![config.command.clone(); config.num_threads]
+                } else {
+                    divide_query_transaction_blocks_tasks(
+                        data,
+                        config.num_threads,
+                        self.get_addresses(),
```

Comment:
Could be convoluted. Also down to just load into a variable in `prepare` and pass that instead of loading into `processor`'s `cache` and then distributing